tuf/api/metadata.py

```diff
@@ -522,21 +521,20 @@ def bump_version(self) -> None:
 class Key:
     """A container class representing the public portion of a Key.
 
-    Please note that "Key" instances are not semanticly validated during
-    initialization: this only happens at signature verification time.
+    Supported key content (type, scheme and keyval) is defined in
+    Securesystemslib.
```

Comment:
```suggestion
    Supported key content (type, scheme and keyval) is defined in
    and validated in Securesystemslib.
```
As you are replacing a sentence about validation it's probably good to add a word for that?